### DISCUSSION TITLE
Send logs to OTEL collector via OTLP

### DISCUSSION THREAD

**Discussion by aliarman1373** [open-telemetry/opentelemetry-collector#6810] [Q&A]
I am trying to send my React app logs to the OTEL collector by loglevel package. Logs are getting sent from the app but the OTEL collector does not receive anything. I should mention that I do not have a problem with sending and receiving traces. The only issue is when I try to send my customized logs.
Here are my configs:
![image](https://user-images.githubusercontent.com/37373915/208017903-09297474-5f17-4c38-946e-8ba04800db74.png)

Docker Compose file:
![image](https://user-images.githubusercontent.com/37373915/208018129-9d001534-cc69-4b4d-a67c-fe1caa0626e4.png)

the log format is json it is like this:

![image](https://user-images.githubusercontent.com/37373915/208018580-e0c4c0d8-fa93-4e04-8cbd-5787ef842024.png)

I am sending logs to "http://localhost:4318/v1/logs" and even when I try that with postman I receive 200 but cannot see logs in any determined exporter and the console log. 

Can someone tell me what is the issue? Thanks :)

**Comment by jpkrohling** [comment]
Do you see any interesting metrics on localhost:8880/metrics ? It should show how many entries the receiver has seen. If it's zero, there might be something wrong with the components before the collector.
